Changes

```diff
@@ -20,6 +20,9 @@ Working version
 
 ### Runtime system:
 
+- #11990: Improve comments and macros around frame descriptors.
+  (Nick Barnes, review by ??)
```

Comment:
That would be me.

Reply:
Note: there is a Changes section `Internal/compiler-libs changes` that I think would be more appropriate here -- to focus the "Runtime system" section on user-visible changes.